docs/api/web-contents.md

```diff
@@ -37,6 +37,12 @@ for all windows, webviews, opened devtools, and devtools extension background pa
 Returns the web contents that is focused in this application, otherwise
 returns `null`.
 
+### `webContents.fromId(id)`
+
+* `id` Integer
+
+Find a `WebContents` instance according to its ID.
```

Comment:
from the instance properties of a an already created WebContents instance, i was not sure to add were to get it, I pretty much copied the description from BrowserWindow.fromId, but do you think i should be specified on both?